layouts/partials/head.html

```diff
@@ -0,0 +1,24 @@
+<meta charset="utf-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1" />
+<title>{{ block "title" . }}
+{{- .Title }} - {{ .Site.Title -}}
```

Comment:
`.Site` can be replaced with `site` (which enables you to avoid trickery like `$.Page.Site.foo`